teammapper-frontend/mmp/src/map/handlers/history.ts

```diff
@@ -26,6 +27,26 @@ export default class History {
         this.snapshots = []
     }
 
+    /**
+     * As snapshots are saved by number (1, 2, 3 etc) and not by node ID, this switches out the number with the affected node ID.
```

Comment:
thanks for the comment. To summarize for me: Now we are converting the numbers to nodeIds as this does not work on the server. But can we change the logic itself to use nodeIds? Do we need to stick with these old numbering system?

Reply:
The numbering system comes from https://www.npmjs.com/package/deep-object-diff, which makes it hard to change unless you want to dive into the package itself.